src/meshapi/tests/test_tasks.py

```diff
@@ -0,0 +1,58 @@
+import os
+from unittest import mock
+
+from django.test import TestCase
+
+from meshapi.tasks import reset_dev_database, run_database_backup
+
+
+# Not intended to test the functionality of, say, dbbackup. More intended
+# to test the environment logic.
+class TestTasks(TestCase):
+    def setUp(self):
+        pass
+
+    @mock.patch("django.core.management.call_command")
+    @mock.patch("meshapi.tasks.MESHDB_ENVIRONMENT", "prod1")
+    def test_run_database_backup(self, mock_call_command_func):
+        os.environ["AWS_ACCESS_KEY_ID"] = "fake"
+        os.environ["AWS_SECRET_ACCESS_KEY"] = "alsofake"
+        self.assertTrue(run_database_backup())
```

Comment:
nit: If you want to avoid returning a value from `run_database_backup()` you can do `mock_call_command_func.assert_called_once()`

Reply:
Yeah but I want to know that it was successful. I suppose instead I could raise an exception, that would work.